eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -43,10 +43,21 @@ jobs:
       - ${{ if not(contains(variables['Build.DefinitionName'], '-pr - ')) }}:
         - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
           parameters:
-              Paths:
-                - "/*"
-                - "!SessionRecords"
-                - "/sdk/${{ parameters.ServiceToTest }}/**/SessionRecords"
+            Paths:
+              - "/*"
+              - "!SessionRecords"
+              - "/sdk/${{ parameters.ServiceToTest }}/**/SessionRecords"
+      - task: DownloadPipelineArtifact@2
```

Comment:
We will want to pass down ProjectListOverrideFileProperty into this template and condition this step on it. Then if it is set to non-empty use `$(${{ parameters.ProjectListOverrideFileProperty }})` for your pattern here and for passing to dotnet test command.

Reply:
Yes, done